StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetAnalyticsHelperTest.swift

```diff
@@ -21,16 +21,7 @@ final class PaymentSheetAnalyticsHelperTest: XCTestCase {
         )
         XCTAssertTrue(STPAnalyticsClient.sharedClient.productUsage.contains("PaymentSheet"))
 
-        _ = PaymentSheet.FlowController(
-            configuration: PaymentSheet.Configuration(),
-            loadResult: .init(
-                intent: .paymentIntent(STPFixtures.paymentIntent()),
-                elementsSession: .makeBackupElementsSession(with: STPFixtures.paymentIntent()),
-                savedPaymentMethods: [],
-                paymentMethodTypes: [.stripe(.card)]
-            ), analyticsHelper: .init(isCustom: true, configuration: PaymentSheet.Configuration())
-        )
-        XCTAssertTrue(STPAnalyticsClient.sharedClient.productUsage.contains("PaymentSheet.FlowController"))
+        // Test for PaymentSheet.FlowController is added to productUsage is in PaymentSheetAPITest.testUpdate()
```

Comment:
It might make more sense to make this call `PaymentSheet.FlowController.create(...)`

Reply:
We could, but create calls load(), which is a bit more difficult to stub out.  We'd need to make these tests network stubbable and/or mocked, which is a bit more work than (imho) is worth to test that product usage is being set.

Reply:
I think we don't even need to wait for the `create` call to complete, we can just call it and forget about it.  No need to stub/mock, is there?

Ooh also just realized, we should clear `STPAnalyticsClient.sharedClient.productUsage` at the start of this test so it fails when it should have.  Right now I guess it's hardly testing anything, just that `STPAnalyticsClient.sharedClient.productUsage` was set at some point in the test suite beforehand 😂

Reply:
Good point on not needing to mock as well as clearing out product usage!